### PR TITLE
Week 1 Turnin

### PR DESCRIPTION
**Pull request by NickShermeister**
Hi. You may notice some unused code. I did a goof somewhere and never figured out where so I commented out the functions and re-wrote them. It's good now.

### CODE REVIEW

**Comment by NickShermeister** [comment]
Commited a final turnin now!

File: gene_finder.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 """
 YOUR HEADER COMMENT HERE
```
Comment (segerphilip) [inline gene_finder.py]: Probably include a comment? Then this would be 🍑y

File: gene_finder.py [hunk 2]
```diff
@@ -30,8 +30,14 @@ def get_complement(nucleotide):
...
     """
-    # TODO: implement this
-    pass
+    if nucleotide == 'A':
+        return 'T'
+    elif nucleotide == 'C':
+        return 'G'
+    elif nucleotide == 'G':
+        return 'C'
+    else:
```
Comment (segerphilip) [inline gene_finder.py]: This is okay, however using elif here for A and then having else catch for wrong input might be good

File: gene_finder.py [hunk 3]
```diff
@@ -45,8 +51,11 @@ def get_reverse_complement(dna):
     >>> get_reverse_complement("CCGCGTTCA")
     'TGAACGCGG'
     """
-    # TODO: implement this
-    pass
+    complementary = ''
+    reversedna = dna[::-1]
```
Comment (segerphilip) [inline gene_finder.py]: 👏 Good function

File: gene_finder.py [hunk 4]
```diff
@@ -62,9 +71,31 @@ def rest_of_ORF(dna):
...
-
+    #TAG, TAA, TGA are the stop codons
+    for n in range(0, (len(dna)-len(dna)%3), 3):
+        if dna[n] in ('T'):
+            codon = dna[n: n + 3]
+            if codon in ('TAG', 'TAA', 'TGA'):
+                return dna[:n]
+    return dna
+    """
+    while nostopcodon:
```
Comment (segerphilip) [inline gene_finder.py]: You should remove comments before doing a PR. It should be in commit history, so no loss.

File: test.py [hunk 5]
```diff
@@ -0,0 +1,78 @@
+def main():
```
Comment (segerphilip) [inline test.py]: So wait, is test.py just for running as a test for gene_finder? Or is it where you were testing how to fix things?

Comment (segerphilip) [inline test.py]: Try to avoid confusion with HEADER COMMENTS!!!

File: gene_finder.py [hunk 6]
```diff
@@ -158,9 +262,20 @@ def gene_finder(dna):
...
+    AAcodes = []
+    print(len(dna))
+    for x in allORFSfinal:
+        if len(x) > threshold:
+            AAcodes.append(coding_strand_to_AA(x))
+    print(len(allORFSfinal))
+    return AAcodes
+
+
+"""if __name__ == "__main__":
```
Comment (segerphilip) [inline gene_finder.py]: Having this uncommented would be fine. Your tests pass, which is also good.

File: test.py (hunk 5 above)
Comment (NickShermeister) [inline test.py]: Yes, Sir.

**Pull request closed without merging**